docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst

```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
 
 .. For details on other ways to authenticate, see LINKTOAUTHHOWTODOCS.
 
+.. _socket_path_issue:
+
+Category "socket_path issue"
+========================================================
+
+**Platforms:** Any
+
+The ``socket_path does not exist or cannot be found``  and ``unable to connect to socket`` message is new in Ansible 2.5. This message means that
```

Comment:
"message is new" -> "messages are new"
"This message means" -> "These messages means"